app/src/main/java/fr/neamar/kiss/forwarder/ExperienceTweaks.java

```diff
@@ -49,6 +49,10 @@ public void run() {
 
     private View mainEmptyView;
     private final GestureDetector gd;
+    private static byte GESTURE_ACTION_SWIPE_UP = 2;       // defaulting to 2 (all apps)
```

Comment:
since those values are not const, I'd rather not use FULL_CAPS for naming -- it looks like a constant